builder/bundles.go

```diff
@@ -378,11 +383,14 @@ func resolvePackagesWithOptions(numWorkers int, set bundleSet, packagerCmd []str
 			// in the actual install to the full chroot.
 			outBuf, errStr := helpers.RunCommandOutputEnv(queryString[0], queryString[1:], []string{"LC_ALL=en_US.UTF-8"})
 			rpm, e := repoPkgFromNoopInstall(outBuf.String())
+			if errStr != nil {
+				log.Debug(log.Dnf, errStr.Error())
+			}
+			log.Debug(log.Dnf, outBuf.String())
+
 			if len(bundle.AllPackages) != 0 && e != nil {
 				e = errors.Wrapf(e, bundle.Name)
-				fmt.Println(e)
-				fmt.Println("error details:")
-				fmt.Println(errStr)
+				log.Error(log.Dnf, e.Error())
```

Comment:
Lets have the two debug statement inside the `if` condition so that we don't clutter the logs with dnf output unless there is an error.
E.g.
```
			if len(bundle.AllPackages) != 0 && e != nil {
				e = errors.Wrapf(e, bundle.Name)
				log.Error(log.Dnf, e.Error())
				log.Debug(log.Dnf, errStr.Error())
				log.Debug(log.Dnf, outBuf.String())
				errorCh <- e
				return
			}
```